common/controllers/framework/framework-step.js

```diff
@@ -13,14 +12,9 @@ class FrameworkStepController extends FormWizardController {
   }
 
   checkEditable(req, res, next) {
-    const isEditable = req.assessment?.isEditable
-    const userPermissions = req.user?.permissions
-    const canEdit = permissionsControllers.check(
-      'person_escort_record:update',
-      userPermissions
-    )
+    const { isEditable, framework } = req.assessment
 
-    if (!isEditable || !canEdit) {
+    if (!isEditable || !req.canAccess(`${snakeCase(framework.name)}:update`)) {
```

Comment:
I can see the need for this here, but I'm not thrilled about the swapping between snake_cake and camelCase throughout the system. However, that's a much bigger issue than this PR, so I guess ... :+1: ?